platforms/r36s/index.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "R36S",
+    "database_key": "r36s",
+    "manufacturer": "-",
```

Comment:
I think the manufacturer is Boyhom

Reply:
I tried to Google it, but did not find any definitive answer. I'll check it out tomorrow too, if I don't find anything, I'll set it to Boyhum.